### PR TITLE
CDK: reorganizing docs out of CDK and into zkEVM where possible

### PR DESCRIPTION
**Pull request by kmurphypolygon**
@EmpieichO Hi Anthony,

Please could you review this PR as it affects zkEVM docs.

We are moving some of the tutorials from CDK in favour of the Kurtosis docs in there which are stable and live.

Please check for CDK mentions in some of the docs I moved into zkEVM as these need to be removed.

Please spin it up and also check the structure and tell me whether you think it works. 

Thank you :) <3

### CODE REVIEW

**Comment by kmurphypolygon** [comment]
Converting to draft as need to decide about validium quickstart and deployment tutorial... delete? or not?

**Comment by kmurphypolygon** [comment]
> Converting to draft as need to decide about validium quickstart and deployment tutorial... delete? or not?

After chatting with @gsilva5 , we're deleting these from live but I pushed them into the dev environment so they're not completely out of view.

**Review by EmpieichO — APPROVED** [review]

File: docs/zkEVM/index.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ hide:
        </a>
     </div>
     <div class="grid-item">
-       <a href="./get-started/deploy-zkevm/">
+       <a href="./get-started/setup-nodes/deploy-zkevm/intro">
           <div class="product-list-item-header">
              <div class="feature-card-heading">Deploy zkEVM</div>
           </div>
-          <p class="feature-paragraph">Get started by deploying zkEVM on Goerli.</p>
+          <p class="feature-paragraph">Get started by deploying zkEVM on Cardona.</p>
```
Comment (kmurphypolygon) [inline docs/zkEVM/index.md]:
```suggestion
          <p class="feature-paragraph">Get started by deploying zkEVM.</p>
```

**Comment by AdityaBirangal** [comment]
@kmurphypolygon @EmpieichO @gsilva5 
OLD cdk guides were very useful. (validium quickstart & all)
Why this  is deleted ? 
You can add 'depreciated' in front of these guides. But directly deleting it suddenly breaks dev flow.

Anyways i have got access by switching to old commit & running docs locally.
But good to have on live site as well with depreciated tag for community.

Thanks :)

**Comment by kmurphypolygon** [comment]
Hi @AdityaBirangal yes, sorry for the inconvenience.

You can still find the docs:

- Validium quickstart here: https://github.com/Snapchain/zkValidium-quickstart - this is also replaced by the Kurtosis guide now: https://docs.polygon.technology/cdk/get-started/quickstart/deploy-stack/
- The rollup quickstart is here: https://docs.polygon.technology/zkEVM/get-started/setup-nodes/local-node/
- The rollup deploy guide is here: https://docs.polygon.technology/zkEVM/get-started/setup-nodes/deploy-zkevm/intro/ (we're fixing this still, should be coming online next week).
- The validium deploy guide is replaced by the Kurtosis build: https://docs.polygon.technology/cdk/get-started/quickstart/deploy-stack/

Redirects should be coming up soon, there is a devops ticket outstanding for those.

Let me know if we can help you with something more. Thank you for the feedback.

**Pull request merged**